### PR TITLE
Increase thresholds for key-funder

### PR DESCRIPTION
**Pull request by nambrot**
### Description

The original values were kind of arbitrarily chosen and generally lead the to the key-funder not giving enough which causes daily transfers. Raising the thresholds to reduce that burden

### CODE REVIEW

**Comment by mattiecnvr** [comment]
Might seriously want to look at adding separate values for RC before we do this.

**Comment by nambrot** [comment]
Good callout, I pushed a change @mattiecnvr

File: typescript/infra/scripts/funding/fund-keys-from-deployer.ts [hunk 1]
```diff
@@ -418,10 +421,15 @@ class ContextFunder {
       desiredBalance,
       'ether',
     );
+    const adjustedDesiredBalance =
+      this.context === Contexts.ReleaseCandidate
+        ? desiredBalanceEther.mul(RC_FUNDING_DISCOUNT)
```
Comment (tkporter) [inline typescript/infra/scripts/funding/fund-keys-from-deployer.ts:426]: This'll throw
```
> n = ethers.BigNumber.from(100)
BigNumber { _hex: '0x64', _isBigNumber: true }
> n.mul(0.2)
Uncaught:
Error: underflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-underflow ] (fault="underflow", operation="BigNumber.from", value=0.2, code=NUMERIC_FAULT, version=bignumber/5.7.0)
```

Something like the NUMERATOR / DENOMINATOR thing we do for the min delta would work well here

Comment (nambrot) [reply typescript/infra/scripts/funding/fund-keys-from-deployer.ts:426]: Great catch!

**Review by tkporter — APPROVED** [review]

**Pull request merged**